### PR TITLE
Permission integration test

### PR DESCRIPTION
**Pull request by loweisz**
# Purpose of PR

## PR Checklist

- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Typescript typings are added/updated/not required

### CODE REVIEW

File: test/cypress/integration/fixtures/user-data.json [hunk 1]
```diff
@@ -2,16 +2,16 @@
   "admin": {
```
Comment (shikaan) [inline test/cypress/integration/fixtures/user-data.json:2]: I think we can delete and ignore this file (gets regenerated on every test run)

File: test/cypress/integration/fixtures/ids-data.json [hunk 2]
```diff
@@ -14,13 +14,15 @@
     "field": "title"
```
Comment (shikaan) [inline test/cypress/integration/fixtures/ids-data.json:14]: Same as the other file

**Review by shikaan — APPROVED** [review]
LGTM only a minor comment

File: test/cypress/integration/fixtures/user-data.json (hunk 1 above)
Comment (loweisz) [reply test/cypress/integration/fixtures/user-data.json]: That's a good point 🙇‍♂️

**Pull request merged**